sentry/src/main/java/io/sentry/util/FileUtils.java

```diff
@@ -0,0 +1,48 @@
+package io.sentry.util;
+
+import java.io.File;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.Nullable;
+
+@ApiStatus.Internal
+public final class FileUtils {
+
+  /**
+   * Deletes the file or directory denoted by a path. If it is a directory, all files and directory
+   * inside it are deleted recursively. Note that if this operation fails then partial deletion may
+   * have taken place.
+   *
+   * @param file file or directory to delete
+   * @return true if the file/directory is successfully deleted, false otherwise
+   */
+  public static boolean deleteRecursively(@Nullable File file) {
+    if (file == null || !file.exists()) {
+      return true;
+    }
+    if (file.isFile()) {
+      return file.delete();
+    }
+    File[] children = file.listFiles();
+    if (children == null) return true;
+    for (File f : children) {
+      if (!deleteRecursively(f)) return false;
+    }
+    return true;
+  }
+
+  /**
+   * If the relative path denotes an absolute path, it is returned back. Otherwise it is returned
+   * the file relative to f For instance, `File.resolve(new File("/foo/bar"), "gav")` is `new
+   * File("/foo/bar/gav")` While `File.resolve(new File("/foo/bar"), "/gav")` is `new File("/gav")`.
+   *
+   * @return concatenated this and [relative] paths, or just [relative] if it's absolute.
+   */
+  public static @Nullable File resolve(@Nullable File f, @Nullable String relative) {
```

Comment:
Not entirely sure why we need that, isn't `File(folder, "name")` enough?

Reply:
yeah, it's just a "safer" method we use in the specto sdk (not only for this method), but maybe it's cleaner to use just `new File(folder, name)` as it's enough